packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -230,12 +246,12 @@ class Radio extends React.Component<Props> {
     };
 
     render(): React.Node {
-        let choices = this.props.choices;
-        let choiceStates;
+        const choices: $ReadOnlyArray<PerseusRadioChoice> = this.props.choices;
+        let choiceStates: $ReadOnlyArray<ChoiceState>;
         if (this.props.static) {
-            choiceStates = _.map(choices, (val) => ({
-                selected: val.correct,
-                crossedOut: val.crossedOut,
+            choiceStates = choices.map((choice) => ({
+                selected: !!choice.correct,
+                crossedOut: false,
```

Comment:
Why can this be changed to false here instead of pulling the value from choice like selected?